packages/dd-trace/test/llmobs/sdk/index.spec.js

```diff
@@ -0,0 +1,1008 @@
+'use strict'
+
+/** Commented-out tests represent features that will be implemented in a later PR */
+
+const { expect } = require('chai')
+const Config = require('../../../src/config')
+
+const LLMObsTagger = require('../../../src/llmobs/tagger')
+const LLMObsEvalMetricsWriter = require('../../../src/llmobs/writers/evaluations')
+const LLMObsAgentProxySpanWriter = require('../../../src/llmobs/writers/spans/agentProxy')
+const LLMObsSpanProcessor = require('../../../src/llmobs/span_processor')
+
+const tracerVersion = require('../../../../../package.json').version
+
+describe('sdk', () => {
+  let LLMObsSDK
+  let llmobs
+  let tracer
+
+  before(() => {
+    tracer = require('../../../../dd-trace')
+    tracer.init({
+      service: 'service',
+      llmobs: {
+        mlApp: 'mlApp',
+        apiKey: 'test'
+      }
+    })
+    llmobs = tracer.llmobs
+
+    // spy on properties
+    sinon.spy(LLMObsSpanProcessor.prototype, 'process')
+    sinon.spy(LLMObsSpanProcessor.prototype, 'format')
+    sinon.spy(tracer._tracer._processor, 'process')
+
+    // stub writer functionality
+    sinon.stub(LLMObsEvalMetricsWriter.prototype, 'append')
+    sinon.stub(LLMObsEvalMetricsWriter.prototype, 'flush')
+    sinon.stub(LLMObsAgentProxySpanWriter.prototype, 'append')
+    sinon.stub(LLMObsAgentProxySpanWriter.prototype, 'flush')
+
+    LLMObsSDK = require('../../../src/llmobs/sdk')
+
+    // remove max listener warnings, we don't care about the writer anyways
+    process.removeAllListeners('beforeExit')
+  })
+
+  afterEach(() => {
+    LLMObsSpanProcessor.prototype.process.resetHistory()
+    LLMObsSpanProcessor.prototype.format.resetHistory()
+    tracer._tracer._processor.process.resetHistory()
+
+    LLMObsEvalMetricsWriter.prototype.append.resetHistory()
+    LLMObsEvalMetricsWriter.prototype.flush.resetHistory()
+
+    LLMObsAgentProxySpanWriter.prototype.append.resetHistory()
+    LLMObsAgentProxySpanWriter.prototype.flush.resetHistory()
+
+    process.removeAllListeners('beforeExit')
+  })
+
+  after(() => {
+    sinon.restore()
+    llmobs.disable()
+    delete global._ddtrace
+    delete require.cache[require.resolve('../../../../dd-trace')]
```

Comment:
Why is this needed?

Reply:
was able to remove it. I had it there because I was having issues between two sets of tests but was able to resolve it